oap-server/server-starter/src/main/resources/ui-initialized-templates/apisix/apisix-service.json

```diff
@@ -50,7 +50,7 @@
             "showYAxis": true
           },
           "expressions": [
-            "meter_apisix_sv_http_status_matched"
+            "aggregate_labels(meter_apisix_sv_http_status_matched,sum(code))"
```

Comment:
I think the hierarchy side metrics should aggravate labels too, since you add `node` label in this metric.